torch/csrc/jit/python/init.cpp

```diff
@@ -734,6 +734,7 @@ void initJITBindings(PyObject* module) {
       return op->schema();
     });
   });
+  m.def("_is_tracing", []() { return jit::tracer::isTracing(); });
```

Comment:
we may already have an api for this, `torch.jit.is_tracing`, or something.

Reply:
I didn't see one in `dir(torch.jit)`